Tests/ActionTests/ActionTests.swift

```diff
@@ -46,15 +46,17 @@ class ActionTests: QuickSpec {
 		describe("action properties") {
 			var inputs: TestableObserver<String>!
 			var elements: TestableObserver<String>!
-			var errors: TestableObserver<ActionError>!
+			var errors: TestableObserver<String>!
```

Comment:
better keep `errors` of type `Error` and assert actual error is thrown instead of asserting strings

Reply:
I couldn't use XCAssertEqual with Error, do you have any ideas?